lab-raziyeh/app/component/navbar/_navbar.scss

```diff
@@ -0,0 +1,21 @@
+@import  "_theme";
```

Comment:
@Raziyehbazargan no need to include the underscore in your SCSS partial imports - this gets implied:  `@import "theme"` works the same